src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.spec.ts

```diff
@@ -40,22 +47,59 @@ describe('CreateAdminComponent', () => {
   });
 
   beforeEach(() => {
+    TestBed.overrideProvider(ActivatedRoute, { useValue: { snapshot: { paramMap: convertToParamMap({ param: adminRole, id: 'id' }) } } });
     fixture = TestBed.createComponent(CreateAdminComponent);
     component = fixture.componentInstance;
-    component.adminFormGroup = new FormGroup({
-      lastName: new FormControl(''),
-      firstName: new FormControl(''),
-      middleName: new FormControl(''),
-      phoneNumber: new FormControl(''),
-      institution: new FormControl(''),
-      email: new FormControl('')
-    });
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('getters', () => {
+    it('should return correct institution form control', () => {
+      expect(component.institutionFormControl).toEqual(component.adminFormGroup.get('institution'));
+    });
+
+    it('should return correct region form control', () => {
+      expect(component.regionFormControl).toEqual(component.adminFormGroup.get('region'));
+    });
+
+    it('should return correct territorial community form control', () => {
+      expect(component.territorialCommunityFormControl).toEqual(component.adminFormGroup.get('territorialCommunity'));
```

Comment:
I really can't note the line you are talking about. Can you please specify the line number? Because it seems pretty valid in the editor.
![image](https://github.com/ita-social-projects/OoS-Frontend/assets/68949838/a64b0632-2dc6-43d1-b9ef-c7f20df24a12)